applications/openshift/worker/file_permissions_worker_kubeconfig/rule.yml

```diff
@@ -8,26 +8,77 @@ platforms:
 {{%- if product == "eks" %}}
 {{%- set octal_perms = "0644" %}}
```

Comment:
the `octal_perms` doesn't seem to be used?

Reply:
Oh - good find. I'll remove that in a subsequent push.